docs/notifications.md

```diff
@@ -64,7 +64,7 @@ omitted if empty).
 
 Recognized stream names:
  * `block_added`
-   Filter: `primary` as an integer with primary (speaker) node index from
+   Filter: `primary` as an byte with primary (speaker) node index from
```

Comment:
Adjust the commit title: we used to see an action in the title, e.g. fix something or implement something or add some functionality. So it's probably better be: `rpc: change the type of BlockFilter's Primary field to byte` or `rpc: change the type of BlockFilter.Primary from int to byte`

Also there's a typo in the commit description: s/`It needs to`/`It's needed to` and s/`to prevent change filter's`/`to prevent changing filters`?